File: judge/fixtures/java/Memory.java
// 판정 fixture: 힙을 계속 붙잡는다. OutOfMemoryError 로 MEMORY_LIMIT 여야 한다.
import java.util.*;
public class Main {
    public static void main(String[] args) {
        List<long[]> keep = new ArrayList<>();
        while (true) keep.add(new long[1 << 20]);
    }
}

File: judge/fixtures/cpp/memory.cpp
// 판정 fixture: 메모리를 계속 붙잡는다. MEMORY_LIMIT 여야 한다.
#include <bits/stdc++.h>
int main() {
    std::vector<std::vector<char>> keep;
    while (true) {
        keep.emplace_back(16 * 1024 * 1024, 1);  // 실제로 써서 페이지를 잡는다
    }
}
